### PR TITLE
Remove portfolio related outputs

### PR DESCRIPTION
**Pull request by jacobvjk**
This PR is a preparation to the general refactoring that will decouple the stress test from PACTA

In this step, the portfolio related parts are removed.
This means that we do not calculate/aggregate/write any results anymore that are used to give a portfolio level result.
In effect, we only retain company trajectories and crispy as outputs.

Additionally, inputs that are only required for portfolio weighting are removed. At this point, this only refers to the sector_exposures.

All functions that exclusively deal with portfolio related calculations/wrangling are dropped too. Similar versions may be needed in the future, when allowing to apply crispy results to a portfolio, but they can easily be obtained again through the git history if need be. Although more streamlined versions may then be a better idea.

I checked that the remaining outputs are identical for both trisk and lrisk between master and this branch.

### CODE REVIEW

**Review by MirijaHa — COMMENTED** [review]
Just  to go sure, did you test that company_trajectories and cripsy_output remain unchanged?

**Comment by jacobvjk** [comment]
> Just to go sure, did you test that company_trajectories and cripsy_output remain unchanged?

Indeed, I checked that `company_trajectories` and `crispy_output` are identical for all asset types and for both trisk and lrisk

**Review by MirijaHa — COMMENTED** [review]
One comment and please note my former comment on the impact of existing results, otherwise 🚀

File: R/aggregate_results.R [hunk 1]
```diff
@@ -21,66 +21,11 @@
...
-
-  company_pd_changes_overall <- company_pd_changes_overall %>%
-    aggregate_pd_change_to_sector_level(horizon = "overall", iter_var = iter_var)
-
   # Aggregate Crispy Results -----
 
   crispy_output <- results_list$company_technology_npv %>%
     dplyr::inner_join(
-      company_pd_changes_overall,
+      results_list$company_pd_changes_overall,
```
[the remark this answers is not in the document]
Comment (jacobvjk) [reply R/aggregate_results.R:28]: correct, but the pd results always also maintained the individual company level results and those were the ones kept in CRISPY, so in effect nothing changes. Which is also why there are no differences for crispy_output despite this input to the join containing less additional information

**Review by MirijaHa — APPROVED** [review]
🧨

**Pull request merged**